snuba/datasets/entity_subscriptions/entity_subscription.py

```diff
@@ -62,9 +62,17 @@ def validate_query(self, query: Union[CompositeQuery[Entity], Query]) -> None:
             NoTimeBasedConditionValidator(entity.required_time_column).validate(query)
 
 
-class SessionsSubscription(EntitySubscriptionValidation, EntitySubscription):
-    MAX_ALLOWED_AGGREGATIONS: int = 2
+class SimpleEntitySubscription(EntitySubscriptionValidation, EntitySubscription):
```

Comment:
This will be controversial, but I don't think you need to change this file at all. The new `Pluggable` class you added can inherit from the `TransactionsSubscription` class and the other one can inherit from `SessionsSubscription` like it does now.

Reply:
Right, that was my initial thought, but the `TransactionsSubscription` is itself the transactions subscriptions entity and the `SessionsSubscription` is itself the sessions subscriptions entity. Which means when we eventually remove these components because they are all in config, then things will break. That's my rational for making two parent classes which all python defined subscriptions entity inherits from. That way, when we eventually delete the child python entity subscriptions, the config ones wont break.